ledger/acctupdates.go

```diff
@@ -1300,7 +1335,7 @@ func (au *accountUpdates) lookupResource(rnd basics.Round, addr basics.Address,
 		// against the database.
 		persistedData, err = au.accountsq.lookupResources(addr, aidx, ctype)
 		if persistedData.round == currentDbRound {
-			if persistedData.addrid != 0 {
+			if err == nil {
```

Comment:
Why is this PR changing lookupResource?

Reply:
Good catch, I don't know how that got in there.